myosuite/envs/myo/assets/hand/reorient_sar.xml

```diff
@@ -0,0 +1,51 @@
+<mujoco model="reorient task">
+    <include file="../../../../simhive/myo_sim/hand/assets/myohand_assets.xml"/>
+    <include file="../../../../simhive/myo_sim/scene/myosuite_scene.xml"/>
+    <compiler meshdir='../../../../simhive/myo_sim/' texturedir='../../../../simhive/myo_sim/'/>
+
+    <default>
+		<default class="reorient">
```

Comment:
Check the formatting here.

Reply:
This is usually when you are mixing tabs and spaces for formatting. Check for that.

Reply:
you are correct, just fixed